testdata/config/basic-http-no-auth.yml

```diff
@@ -10,7 +10,7 @@ debugging:
 integrations:
   - type: git
     poll_interval: 1h
-    remote: https://github.com/fossas/broker.git
```

Comment:
I changed the remote because the integration test: `with_successful_http_no_auth_integration()` in fix.rs wasn't working as intended. 

I talked to scott and he told me that it is because broker removes PATH field when using fossa analyze through broker. The results from fossa analyze and broker would differ when running a scan on the repo:  https://github.com/fossas/broker.git  due to static analysis. Therefore I changed the remote that we are scanning.